cpg_utils/config-template.toml

```diff
@@ -163,6 +163,7 @@ vep ='vep:105'
 verify-bam-id = 'verify-bam-id:1.0.1'
 multiqc = 'multiqc:v1.12'
 fastqc = 'fastqc:v0.11.9_cv8'
+happy-vcfeval = 'happy-vcfeval:1.0'
```

Comment:
Is this something we want in the generic template or an addition in the AIP configuration?

Note that you can specify multiple config files and they get merged in order.

Reply:
My main thought was that the image will initially be used in working out how to run the validation, but as the validation stack can be dropped into any cohort, the QC stage may become a standard post-joint-call main pipeline job in future.

If there's substantial benefit in normalising variants, vcfeval could also become a non-validation part of the pipeline.

Happy to use a minimal config to update

Reply:
Sounds good to me if you think this is more generally useful in the future!